### PR TITLE
Minor updates

### PR DESCRIPTION
**Pull request by newpavlov**
I haven't added `RETRYABLE` constant and `is_retryable` method, as they can be added later in a backwards compatible way if need for them will arise.

### CODE REVIEW

File: src/error.rs [hunk 1]
```diff
@@ -13,31 +13,37 @@ use core::fmt;
 use std::{io, error};
 
 // A randomly-chosen 16-bit prefix for our codes
-pub(crate) const CODE_PREFIX: u32 = 0x57f40000;
+pub(crate) const CODE_PREFIX: u32 = 0x57f4c500;
```
Comment (dhardy) [inline src/error.rs]: Apparently this is now a 24-bit prefix.

I was thinking of allowing an 8-bit section per platform for platform-specific constants, hence leaving 16-bits zero, but I guess it's not too important.

File: src/error.rs [hunk 2]
```diff
@@ -13,31 +13,37 @@ use core::fmt;
...
 #[derive(Copy, Clone, Eq, PartialEq)]
 pub struct Error(NonZeroU32);
 
 impl Error {
+    /// An unknown error.
+    /// 
+    /// This is the following constant: 57f4c500 (hex) / 1475659008 (decimal).
+    ///
+    /// The constant value is provided for reference only. It may change in
+    /// future and it will not be considered a breaking change.
```
Comment (dhardy) [inline src/error.rs]: I think post-release we should try not to change this though?

File: src/lib.rs [hunk 3]
```diff
@@ -67,12 +67,12 @@
...
 //! We always choose failure over returning insecure "random" bytes. In general,
-//! on supported platforms, failure is unlikely, though not impossible. If an
-//! error does occur, then it is likely that it will occur on every call to
+//! on supported platforms, failure is highly unlikely, though not impossible.
+//! If an error does occur, then it is likely that it will occur on every call to
 //! `getrandom`, hence after the first successful call one can be reasonably
 //! confident that no errors will occur.
 //! 
-//! On unsupported platforms, `getrandom` always fails.
+//! On unsupported platforms, `getrandom` always fails with [`Error::UNKNOWN`].
```
Comment (dhardy) [inline src/lib.rs]: wrong error

File: src/error.rs (hunk 2 above)
Comment (newpavlov) [reply src/error.rs]: Yes, but I think we should specify somehow, that code should not match on the given error code and use constant instead. I guess we can change wording somehow, but I don't have good ideas right now.

File: src/error.rs (hunk 1 above)
Comment (newpavlov) [reply src/error.rs]: Hm, platform specific codes are defined by platform and not by us, so I am not sure how it would work.

**Comment by newpavlov** [comment]
>BTW I think that would be spelt retriable.

Looks like both variants can be used: https://english.stackexchange.com/questions/305273

File: src/error.rs (hunk 1 above)
Comment (dhardy) [reply src/error.rs]: Not existing platform-specific codes, but at least the wasm-stdweb impl seems to want its own codes.

Comment (newpavlov) [reply src/error.rs]: Hm, how about simply listing those platform-specific error codes in documentation?

**Pull request merged**